SetupDataPkg/ConfApp/SetupConf.c

```diff
@@ -599,59 +496,42 @@ ProcessDfciUsbInput (
 {
   EFI_STATUS  Status;
   CHAR16      *FileName;
-  CHAR16      *FileName2;
   CHAR8       *XmlString;
   UINTN       XmlStringSize;
 
-  DfciUiExitSecurityBoundary ();
-
   FileName  = NULL;
   XmlString = NULL;
 
   //
-  // Process request
+  // Process request, from the DEFAULT_USB_FILE_NAME
   //
-  Status = BuildUsbRequest (L".svd", &FileName);
-  if (EFI_ERROR (Status)) {
-    DEBUG ((DEBUG_ERROR, "Error building Usb Request. Code=%r\n", Status));
-  } else {
+  FileName = AllocateCopyPool (sizeof (DEFAULT_USB_FILE_NAME), DEFAULT_USB_FILE_NAME);
+  if (NULL != FileName) {
     Status = DfciRequestJsonFromUSB (

```

Comment:
Do we still need this Dfci named function?

Reply:
Nope, this is updated to use SVD *.